frontend/app/chat/page.tsx

```diff
@@ -1,85 +1,100 @@
           <ChatMessages history={history} />
         </Card>
-        <Card className="fixed left-1/2 w-full max-w-3xl bg-gray-100 dark:bg-gray-800 -translate-x-1/2 bottom-16 px-5 py-5 mb-5">
+        <Card className="md:fixed md:rounded md:left-1/2 w-full max-w-3xl bg-gray-100  dark:bg-gray-800 md:-translate-x-1/2 md:bottom-16 px-5 py-5 md:mb-5 hover:shadow-none shadow-none">
           <form
-            onSubmit={(e) => {
-              e.preventDefault();
-              if (!isPending) askQuestion();
+            onSubmit={e => {
+              e.preventDefault()
+              if (!isPending) askQuestion()
             }}
-            className="w-full flex items-center justify-center gap-2"
-          >
-            <textarea
-              autoFocus
-              value={question}
-              onChange={(e) => setQuestion(e.target.value)}
-              onKeyDown={(e) => {
-                if (e.key === "Enter" && !e.shiftKey) {
-                  e.preventDefault(); // Prevents the newline from being entered in the textarea
-                  if (!isPending) askQuestion(); // Call the submit function here
-                }
-              }}
-              className="w-full p-2 border border-gray-300 dark:border-gray-500 outline-none rounded dark:bg-gray-800"
-              placeholder="Begin conversation here..."
-            />
-            <Button type="submit" isLoading={isPending}>
-              {isPending ? "Thinking..." : "Chat"}
-            </Button>
-            {/* Reset Button */}
-            <Button
-              className="px-3"
-              variant={"tertiary"}
-              type="button"
-              onClick={resetHistory}
-              disabled={isPending}
-            >
-              <MdAutorenew className="text-2xl" />
-            </Button>
-            {/* Mic Button */}
-            <Button
-              className="px-3"
-              variant={"tertiary"}
-              type="button"
-              onClick={startListening}
-              disabled={!speechSupported}
-            >
-              {isListening ? (
-                <MdMicOff className="text-2xl" />
-              ) : (
-                <MdMic className="text-2xl" />
-              )}
-            </Button>
-            <Link href={"/config"}>
-              <Button className="px-3" variant={"tertiary"}>
-                <MdSettings className="text-2xl" />
+            className="w-full flex flex-col md:flex-row items-center justify-center gap-2 ">
+            <div className="flex gap-1 w-full">
+              <input
+                autoFocus
+                value={question}
+                type="text"
+                onChange={e => setQuestion(e.target.value)}
+                onKeyDown={e => {
+                  if (e.key === "Enter" && !e.shiftKey) {
+                    e.preventDefault() // Prevents the newline from being entered in the textarea
+                    if (!isPending) askQuestion() // Call the submit function here
```

Comment:
Since the input is no longer a `<textarea/>`, the condition `!e.shiftKey` seems deprecated